### DISCUSSION TITLE
Enable cron for Druid reports

### DISCUSSION THREAD

**Discussion by Venkatesh-Dayani23** [Sunbird-Obsrv/Community#165] [Issues] [answered]
Hi Team,
Currently we are using 5.0.0 we are running this **run-report** Jenkins job manually for multiple jobs to get the druid reports.Can you kindly help us to automate this job.

**Comment by prashanthShiksha** [comment]
@sowmya-dixit

**Comment by sowmya-dixit** [comment, marked as answer]
@Venkatesh-Dayani23 These reports execution can be automated. There is AdhocScripts setup in Obsrv which submits these reports to execute based on report_schedule configuration. Please have below setup done and make sure its working fine.

1. Have AdhocScripts is build & deployed - `Deploy/DataPipeline/AdhocScripts` deployed with `common` as tag
2. Ensure `submit-all-jobs` & `start-jobmanager` cron setup is done on spark machine. - Verify if these cron setup jobs are running fine.

Note: Cron setup jobs can be verified once by running manually using same commands used in cron setup.

Let us know if you face any challenges in setting up these.

**Reply by Venkatesh-Dayani23** [reply]
Hi @sowmya-dixit 

We have built and deployed AdhocScripts with common tag. Submit-all-jobs and start-job-manager is already setup in Spark machine. We ran one of the job from the cron setup manually it is running fine. Kindly let us know what to do further.

**Reply by sowmya-dixit** [reply]
@Venkatesh-Dayani23 
Have you verified any report execution from the cron setup? If the reports are running correctly with the cron setup, they will be executed as per report schedule. There's nothing more to do except to confirm the report executions via the Spark server logs or by checking the report files on the Blob store.
